azure-pipelines.yml

```diff
@@ -18,81 +16,18 @@ trigger:
+pool:
+  vmImage: ubuntu-latest
 
-      - script: |
-          set -Eeuo pipefail
-
-          if [[ -z ${BASE_TAG} ]] ; then
-            export BASE_TAG=$(cat keboola.docker-custom-python.latest-build/base-python-artifact)
-          fi
-          rm -rf keboola.docker-custom-python.latest-build
-          printf "Base Tag: $BASE_TAG\n"
-          export BASE_IMAGE=keboola.azurecr.io/docker-custom-python:$BASE_TAG
-          printf "Base Image: $BASE_IMAGE\n"
-          ./create_dockerfile.sh
-          echo "##vso[task.setvariable variable=BASE_TAG;isOutput=true]$BASE_TAG"
-        displayName: Get Latest Tag and create dockerfile
-        name: get_build
-        env:
-          BASE_TAG: '$(BASE_TAG)'
-
-      - task: DockerCompose@0
-        inputs:
-          containerregistrytype: 'Azure Container Registry'
-          azureSubscription: 'ACR'
-          azureContainerRegistry: '{"loginServer":"keboola.azurecr.io", "id" : "/subscriptions/951eaeee-8981-4b7b-b1fd-12145a2fa848/resourceGroups/keboola-acr-prod-rg/providers/Microsoft.ContainerRegistry/registries/keboola"}'
-          dockerComposeFile: '**/docker-compose.yml'
-          dockerComposeCommand: 'build'
-          action: 'Run a Docker Compose command'
-        displayName: 'Build the image'
-
-      - script: docker-compose run --rm --entrypoint python dev -m unittest tests/transformationTest.py
-        displayName: 'Run tests'
-
-      - script: |
-          docker tag keboola/python-transformation-v2 $(APP_IMAGE):latest
-          docker tag keboola/python-transformation-v2 $(APP_IMAGE):$(imageTag)
-        condition: and(succeeded(), eq(variables['isTaggedBuild'], 'true'))
-        displayName: 'Tag the build'
-
-      - task: Docker@2
-        inputs:
-          containerRegistry: $(azureContainerRegistry)
-          repository: $(imageRepository)
-          command: 'push'
-          tags: |
-            latest
-            $(imageTag)
-        condition: and(succeeded(), eq(variables['isTaggedBuild'], 'true'))
-        displayName: 'Push to ACR'
-
-      - script: ./deploy.sh
-        env:
-          APP_IMAGE: $(APP_IMAGE)
-          KBC_DEVELOPERPORTAL_APP: $(KBC_DEVELOPERPORTAL_APP)
-          KBC_DEVELOPERPORTAL_PASSWORD: $(KBC_DEVELOPERPORTAL_PASSWORD)
-          KBC_DEVELOPERPORTAL_USERNAME: $(KBC_DEVELOPERPORTAL_USERNAME)
-          KBC_DEVELOPERPORTAL_VENDOR: $(KBC_DEVELOPERPORTAL_VENDOR)
-          APP_TAG: $(imageTag)
-        condition: and(succeeded(), eq(variables['isTaggedBuild'], 'true'))
-        displayName: "Deploy to Dev Portal"
+jobs:
+  - template: azure-pipelines/jobs/build-transformation-image.yml
+    parameters:
+      displayName: Build Python Transformation
+      componentName: python-transformation-v2
+      baseImageTagPrefix: python-3.10-
+
+  - template: azure-pipelines/jobs/build-transformation-image.yml
+    parameters:
+      displayName: Build Snowpark Transformation
+      componentName: python-snowpark-transformation
```

Comment:
I thought the component ID was going to be just `keboola.snowpark-transformation`.  maybe we should confirm it?

Reply:
Well we have `keboola.python-mlflow-transformation`, so I figured out it's gonna be `python-snowpark-transformation`. But I have checked DBX now and it's `keboola.databricks-transformation`, so obivously both variants are used :smile: I have no strong preference, do I'll ask Chocho probably.

Reply:
But.. Snowpark it not just a Python thing, there's also Java & Scala SDK available, so I guess we should keept the `python` word in the name.

Reply:
yeah, that's a good point